Branch-SDK/src/io/branch/referral/util/BranchEvent.java

```diff
@@ -280,6 +377,33 @@ protected boolean shouldUpdateLimitFacebookTracking() {
         public boolean shouldRetryOnFail() {
             return true; // Branch event need to be retried on failure.
         }
+
+        /**
+         * Some V2 Standard Events need to have their BUO redacted.
+         * @return {code true} if the Event BUO should be redacted.
+         */
+        private boolean redactBuo() {
```

Comment:
This is not really your thing, but this whole BUO redaction kinda makes me scratch my head. My question as a layperson would be, "Why let me attach a BUO in the first place then?" "Why doesn't it show up on my dashboard if I attached it?"

Reply:
This is a great question.  The PRD was very specific on this though, and a review of that document should be done to make sure that we are progressing in the right direction.